src/components/dev-hub/select.js

```diff
@@ -45,7 +45,7 @@ const Options = styled('ul')`
         narrow
             ? `${OPTIONS_POSITION_OFFSET_NARROW}px`
             : `${OPTIONS_POSITION_OFFSET}px`};
-    width: 100%;
+    width: calc(100% + ${2 * BORDER_SIZE}px);
```

Comment:
The dropdown did not seem to expand all the way across the component. I'll look into other select uses to make sure this aligns.